Add a HTTP endpoint taking a Kuzzle API request in JSON format (#1831)
kuzzleio/kuzzle: doc/2/api/essentials/query-syntax/index.md

- `mimetype`: MIME type
- `file`: file content, encoded in base64

### JSON query endpoint

Kuzzle also exposes an endpoint to send queries using the standard JSON request format used by other protocols.  

> This makes it possible to avoid with the use of HTTP routes and to send queries via the HTTP protocol in the same way as for other protocols.

Reviewer: ```suggestion
This makes it possible to avoid the use of the REST API and to send queries via the HTTP protocol the same way as for any other protocols.
```

This endpoint is accessible on the route `POST http://<host>:<port>/_query`:

```bash
$ curl -X POST -H  "Content-Type: application/json" "http://localhost:7512/_query" --data '{
  "controller":"server", 
  "action":"now" 
}'
```

The body of the query will be processed by Kuzzle as a standard query.

::: warning

> This endpoint does not allow to benefit from the advantages of the cache system integrated to HTTP via URLS.

Reviewer: ```suggestion
This endpoint does not allow to benefit from the advantages of the cache system integrated to HTTP via URLs.
```
